www-client/otter-browser/otter_browser-0.9.12.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="A lightweight QtWebKit browser"
+DESCRIPTION="Project aiming to recreate the best aspects of the \
+classic Opera (12.x) UI using Qt5."
+HOMEPAGE="http://otter-browser.org/"
+COPYRIGHT="2013-2017 Otter Browser Team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/OtterBrowser/otter-browser/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="1a184561b9dbf3443cf1f83ad8604976b1b06cc00a93306331b364f237363e20"
+ADDITIONAL_FILES="otter.rdef.in"
+SOURCE_DIR="otter-browser-$portVersion"
```

Comment:
Please swap SOURCE_DIR and ADDITIONAL_FILES.
Source: [HaikuPorter guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)